runtime/doc/options.txt

```diff
@@ -3804,9 +3803,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 	`:lgrepadd`, `:cfile`, `:cgetfile`, `:caddfile`, `:lfile`, `:lgetfile`,
 	and `:laddfile`.
 
-	This would be mostly useful when you use MS-Windows.  If |+iconv| is
-	enabled and GNU libiconv is used, setting 'makeencoding' to "char" has
-	the same effect as setting to the system locale encoding.  Example: >
+	This would be mostly useful when you use MS-Windows.  If GNU libiconv
+	is used, setting 'makeencoding' to "char" has the same effect as
+	setting to the system locale encoding.  Example: >
```

Comment:
hmm, maybe "system encoding" actually is UTF-8 here (even when someone insists of using a non-UTF8 system)

Reply:
I don't have time right now but I'll look into it, thanks for the heads up

Reply:
I'll leave it as it was as I won't have time to dig into that anytime soon